core/consensus/sumeragi.cpp

```diff
@@ -165,16 +165,11 @@ namespace sumeragi {
             update();
         }
 
-        Context(std::vector<std::unique_ptr<peer::Node>>&& peers)
-        {
-            for (auto&& p : peers) {
-                validatingPeers.push_back(std::move(p));
-            }
-        }
-
         void update()
         {
+            logger::debug("sumeragi") << "Context update!";
             auto peers = config::PeerServiceConfig::getInstance().getPeerList();
```

Comment:
This is minor refactor, but I think following code is more safe:
```
validatingPeers.clear();

{
  auto peers = config::PeerServiceConfig::getInstance().getPeerList();
  for ( auto&& p : peers ) {
    validatingPeers.push_back( std::move(p) );
  }
}
```
It is clear that peer is not used after move semantics works.